### PR TITLE
Theoretical background

### PR DESCRIPTION
**Pull request by adamkucharski**
* **Please check if the PR fulfills these requirements**

- [X ] I have read the CONTRIBUTING guidelines 
- [ X] A new item has been added to `NEWS.md`. *This is just a vignette, but can add if required.*
- [X ] Tests for the changes have been added (for bug fixes / features). **This is just a vignette.**
- [X ] Docs have been added / updated (for bug fixes / features). **This is adding a vignette.**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Add vignette on theoretical background. This is currently a draft to allow input from @BlackEdder before converting to full PR.

### CODE REVIEW

File: vignettes/theoretical_background.Rmd [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+This equation doesn't have an analytical solution, but can be solved numerically with Newton’s method to find the final epidemic size, i.e. the unique solution in (0,1]:
+
+1. Select an arbitrary initial estimate $x_0$ and tolerance $T$ ;
+2. Solve $\frac{dF}{d \phi} (x_0) \Delta x = −F(x_0) ~ \text{for } ~ \Delta x$ ;
+3. Set $x_{i+1} = x_i + \Delta x$;
+4. Repeat until $x_i$ converges sufficiently (i.e. difference in each step is less than $T$)
+
+## Varying susceptibility
+
+We can extend the above formulation to account for variable susceptibility among individuals, following the methods outlined in [Miller (2012)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3506030/). First, we define $p(x)$ to be the probability density for a randomly chosen individual to be of suceptibility type $x$. If we define the probability an individual of type $x$ to be ultimately infected as $\phi(x)$, then the overall proportion of the population to be infected is defined as follows, depending on whether $p(x)$ is a continuous or discrete distribution:
```
Comment (BlackEdder) [inline vignettes/theoretical_background.Rmd:60]: Spelling of (2nd) suceptibility -> susceptibility

Comment (adamkucharski) [reply vignettes/theoretical_background.Rmd]: Now fixed

File: vignettes/theoretical_background.Rmd [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+\hat\phi = \int_0^\infty \phi(x) p(x) dx \\
+\text{or} \\
+\hat\phi = \sum_x \phi(x) p(x) dx \\
+$$
+
+We therefore obtain the following relationship between $\phi$ and $\hat\phi$, under the assumption that $S(x,0)$ is near 1 initially:
+
+$$
+1-\phi(x) = S(x,0) e^{-R_0 \hat\phi } \\
+\hat\phi = 1- \sum_x p(x) e^{-R_0 \hat\phi }
```
[the remark this answers is not in the document]
Comment (adamkucharski) [reply vignettes/theoretical_background.Rmd]: Thanks. Have pushed an expanded version and will convert to a full PR.

**Comment by pratikunterwegs** [comment]
Just nudging @BlackEdder to take a look at this final size theory PR, thanks!

**Comment by BlackEdder** [comment]
Thanks, I'll take a quick look tomorrow morning

File: vignettes/theoretical_background.Rmd [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+We can extend the above formulation to account for variable susceptibility among individuals, following the methods outlined in [Miller (2012)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3506030/). First, we define $p(x)$ to be the probability density for a randomly chosen individual to be of susceptibility type $x$. If we define the probability an individual of type $x$ to be ultimately infected as $\phi(x)$, then the overall proportion of the population to be infected is defined as follows, depending on whether $p(x)$ is a continuous or discrete distribution:
+
+$$
+\hat\phi = \int_0^\infty \phi(x) p(x) dx \\
+\text{or} \\
+\hat\phi = \sum_x \phi(x) p(x) dx \\
+$$
+
+If we define $\hat\phi$ to be the expected number of transmissions to a given individual of type $x$, we can therefore calculate the probability that this individual remains susceptible. This gives the following relationship between $\phi$ and $\hat\phi$:
```
[the remark this answers is not in the document]
Comment (pratikunterwegs) [reply vignettes/theoretical_background.Rmd]: Resolved in https://github.com/epiverse-trace/finalsize/pull/193/commits/305de7392e32dadf2b6eff8ae6fc3870451b62eb

**Pull request merged**